### PR TITLE
Added Bing Layer support and made it compatible with Leaflet 1.0 

### PR DESCRIPTION
**Pull request by octavm**

### CODE REVIEW

File: g-leaflet-shramov/pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <dependency>
             <groupId>org.peimari</groupId>
             <artifactId>g-leaflet</artifactId>
-            <version>0.4.11</version>
+            <version>1.0.0-b2</version>
```
Comment (mstahv) [inline g-leaflet-shramov/pom.xml:26]: There is already 1.0.0-b3, with your awesome improvements, maybe build against that?

Comment (mstahv) [inline g-leaflet-shramov/pom.xml:26]: Oops, ignore that, g-leaflet is still in b2 :-)

**Comment by octavm** [comment]
Could we get a release via Vaadin Directory with these changes ? Thanks.

**Comment by mstahv** [comment]
Sure, I'll make some tests and build one right now.

**Comment by mstahv** [comment]
0.0.4 now available, let me know if there are some issues with the release.

**Pull request merged**